cpp/include/cudf/filling.hpp

```diff
@@ -203,5 +203,35 @@ std::unique_ptr<column> sequence(
   scalar const& init,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
+/**
+ * @brief Generate a sequence of timestamps of `n` rows, increment by calendrical months
+ *
+ * The ith row in the output column `output[i]` is i * `months` away from the `init`
+ * timestamp (i is zero-indexed). If the given new date is invalid, the date is scaled
+ * back to the last available day of that month.
```

Comment:
I think this is a bit more concise and precise. Though, need to define what "invalid" date means. 
```suggestion
 * @brief Generate a sequence of timestamps beginning at `init` and incrementing by `months` for each successive element, i.e., `output[i] = init + i * months` 
 *
 * If a given date is invalid, the date is scaled back to the last available day of that month.
```